error when using `oh-my-posh font install Meslo`

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

```
oh-my-posh font install Meslo

Caught panic:

runtime error: invalid memory address or nil pointer dereference

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
        C:/hostedtoolcache/windows/go/1.24.1/x64/src/runtime/debug/stack.go:26 +0x5e
runtime/debug.PrintStack()
        C:/hostedtoolcache/windows/go/1.24.1/x64/src/runtime/debug/stack.go:18 +0x13
github.com/charmbracelet/bubbletea.(*Program).recoverFromPanic(0xc0001483c0, {0x113c6c0, 0x1a2e180})
        C:/Users/runneradmin/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.3.5/tea.go:810 +0xac
github.com/charmbracelet/bubbletea.(*Program).Run.func2()
        C:/Users/runneradmin/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.3.5/tea.go:601 +0xe8
panic({0x113c6c0?, 0x1a2e180?})
        C:/hostedtoolcache/windows/go/1.24.1/x64/src/runtime/panic.go:792 +0x132
github.com/jandedobbeleer/oh-my-posh/src/font.(*main).Init(0xc00012c360)
        D:/a/oh-my-posh/oh-my-posh/src/font/cli.go:192 +0x63f
github.com/charmbracelet/bubbletea.(*Program).Run(0xc0001483c0)
        C:/Users/runneradmin/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.3.5/tea.go:648 +0x83c
github.com/jandedobbeleer/oh-my-posh/src/font.Run({0xc00000a1b0, 0x5}, {0x14048c0, 0xc0001141c0}, 0x1, {0x1215321, 0x1})
        D:/a/oh-my-posh/oh-my-posh/src/font/cli.go:379 +0x117
github.com/jandedobbeleer/oh-my-posh/src/cli.init.func12(0xc0001a4f00?, {0xc000114180?, 0x4?, 0x1215eee?})
        D:/a/oh-my-posh/oh-my-posh/src/cli/font.go:56 +0x1b1
github.com/spf13/cobra.(*Command).execute(0x1a50c60, {0xc000114120, 0x2, 0x2})
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1019 +0xa7b
github.com/spf13/cobra.(*Command).ExecuteC(0x1a50f20)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1148 +0x40c
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1071
github.com/jandedobbeleer/oh-my-posh/src/cli.Execute()
        D:/a/oh-my-posh/oh-my-posh/src/cli/root.go:93 +0x1a
main.main()
        D:/a/oh-my-posh/oh-my-posh/src/main.go:8 +0xf
```

This works fine:
```
oh-my-posh font install
# select Meslo, press [enter], font is installed
```

```
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.5.1
PSEdition                      Core
GitCommitId                    7.5.1
OS                             Microsoft Windows 10.0.22631
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

oh-my-posh --version
25.21.0
```



### Theme

default config

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 25.21.0

Shell: pwsh (7.5.1)

Prompt:

 xxx   ~                                                                                   in pwsh at 11:52:02

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   0 ms
Path(true)                                -   2 ms
Git(false)                                -   4 ms
Root(true)                                -   0 ms
Status(true)                              -   1 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 7.442ms

Cache path: C:\Users\xxx\AppData\Local\oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[DEBUG] 11:52:02.720 debug.go:36 → debug mode enabled
[TRACE] 11:52:02.721 load.go:Path() - 0s
[DEBUG] 11:52:02.721 load.go:loadConfig:122 → no config file specified, using default
[TRACE] 11:52:02.721 load.go:loadConfig() - 0s
[TRACE] 11:52:02.721 load.go:Load() - 0s
[DEBUG] 11:52:02.721 terminal.go:Init:54 → plain mode enabled
[TRACE] 11:52:02.721 path.go:Path() - 0s
[DEBUG] 11:52:02.721 file.go:Init:26 → loading cache file: C:\Users\xxx\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 11:52:02.722 file.go:Init:47 → loading cache key: https://github.com/ryanoasis/nerd-fonts/releases/download/v3.4.0/Meslo.zip
[DEBUG] 11:52:02.722 file.go:Init:47 → loading cache key: font_list_cache
[TRACE] 11:52:02.722 file.go:Init(C:\Users\xxx\AppData\Local\oh-my-posh\omp.cache) - 531.5µs
[TRACE] 11:52:02.722 path.go:Path() - 0s
[DEBUG] 11:52:02.722 file.go:Init:26 → loading cache file: C:\Users\xxx\AppData\Local\oh-my-posh\omp.cache.1d5d0997-d339-4745-b4bb-bfbe770dc541
[DEBUG] 11:52:02.722 file.go:Init:47 → loading cache key: prompt_count_cache
[DEBUG] 11:52:02.722 file.go:Init:47 → loading cache key: template_cache
[TRACE] 11:52:02.722 file.go:Init(C:\Users\xxx\AppData\Local\oh-my-posh\omp.cache.1d5d0997-d339-4745-b4bb-bfbe770dc541) - 0s
[DEBUG] 11:52:02.722 file.go:Get:75 → getting cache key: prompt_count_cache with value: 5
[TRACE] 11:52:02.722 terminal.go:setPromptCount() - 0s
[TRACE] 11:52:02.722 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.722 terminal.go:setPwd:115 → C:\Users\xxx
[TRACE] 11:52:02.722 terminal.go:setPwd() - 0s
[TRACE] 11:52:02.722 terminal.go:Init() - 531.5µs
[TRACE] 11:52:02.722 terminal.go:Shell() - 0s
[TRACE] 11:52:02.722 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.722 file.go:Get:75 → getting cache key: template_cache with value: {"SegmentsCache":{"Path":{"Segment":{"Text":"  \u003cLINK\u003efile:C:/Users/xxx\u003cTEXT\u003e~\u003c/TEXT\u003e\u003c/LINK\u003e ","Index":1},"Location":"C:/Users/xxx","Path":"~","Folders":[],"StackCount":0,"Writable":true,"RootDir":false},"Root":{"Segment":{"Text":"  ","Index":2}},"Session":{"Segment":{"Text":" xxx ","Index":0},"SSHSession":false},"Shell":{"Segment":{"Text":"in \u003cp:blue\u003e\u003cb\u003epwsh\u003c/b\u003e\u003c/\u003e ","Index":0},"Name":"pwsh","Version":"7.5.1"},"Status":{"Segment":{"Text":"  ","Index":3},"String":"0","Meaning":"0","Error":false},"Time":{"Segment":{"Text":"at \u003cp:blue\u003e\u003cb\u003e11:50:29\u003c/b\u003e\u003c/\u003e","Index":1},"CurrentDate":"2025-05-08T11:50:29.6751165+02:00","Format":"15:04:05"}},"Segments":{},"Var":{},"PWD":"~","Folder":"xxx","PSWD":"","UserName":"xxx","HostName":"DESKTOP-DgDpSnL","ShellVersion":"7.5.1","Shell":"pwsh","AbsolutePWD":"C:\\Users\\xxx","OS":"windows","Version":"25.21.0","PromptCount":5,"SHLVL":0,"Jobs":0,"Code":0,"WSL":false,"Root":true}
[TRACE] 11:52:02.722 cache.go:restoreCache() - 0s
[DEBUG] 11:52:02.722 writer.go:Init:106 → terminal program: Windows Terminal
[DEBUG] 11:52:02.722 writer.go:Init:107 → terminal shell: shell
[DEBUG] 11:52:02.722 terminal.go:Getenv:77 → NO DATA
[TRACE] 11:52:02.722 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[DEBUG] 11:52:02.722 file.go:Get:70 → cache key not found: accent_color
[TRACE] 11:52:02.722 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[DEBUG] 11:52:02.722 terminal_windows.go:WindowsRegistryKeyValue:198 → ColorizationColor(DWORD): 0xC40078D4
[TRACE] 11:52:02.722 colors_windows.go:GetAccentColor() - 531.9µs
[TRACE] 11:52:02.722 colors.go:SetAccentColor() - 531.9µs
[TRACE] 11:52:02.722 terminal.go:Shell() - 0s
[DEBUG] 11:52:02.722 terminal.go:Getenv:77 → 7.5.1
[TRACE] 11:52:02.722 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 11:52:02.722 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 11:52:02.722 text.go:patchTemplate:125 → {{ .Shell }} in {{ .Folder }}
[TRACE] 11:52:02.722 text.go:Render({{ .Shell }} in {{ .Folder }}) - 0s
[TRACE] 11:52:02.722 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 terminal.go:Getenv:77 → 30
[TRACE] 11:52:02.723 terminal.go:Getenv(POSH_CURSOR_LINE) - 531µs
[DEBUG] 11:52:02.723 terminal.go:Getenv:77 → 1
[TRACE] 11:52:02.723 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.723 segment.go:Execute:112 → segment: Status
[DEBUG] 11:52:02.723 segment.go:Execute:112 → segment: Session
[DEBUG] 11:52:02.723 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.723 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.723 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.723 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.723 terminal.go:Getenv:77 → NO DATA
[DEBUG] 11:52:02.723 properties.go:GetString:28 → status_template: {{ .Code }}
[TRACE] 11:52:02.723 terminal.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 11:52:02.723 terminal.go:Getenv:77 → NO DATA
[TRACE] 11:52:02.723 terminal.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.723 segment.go:Execute:112 → segment: Root
[DEBUG] 11:52:02.723 file.go:Get:70 → cache key not found: toggle_cache
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 segment.go:Execute:112 → segment: Path
[DEBUG] 11:52:02.723 file.go:Get:70 → cache key not found: toggle_cache
[TRACE] 11:52:02.723 terminal_windows.go:Root() - 0s
[DEBUG] 11:52:02.723 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.723 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 separator.go:Separator() - 0s
[TRACE] 11:52:02.723 terminal.go:Shell() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.723 text.go:patchTemplate:125 →  {{ if .Data.SSHSession }} {{ end }}{{ .UserName }}
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.723 segment.go:Execute:112 → segment: Git
[DEBUG] 11:52:02.723 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.723 segment.go:isToggled:229 → no toggles found
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.723 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 11:52:02.723 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 11:52:02.723 text.go:patchTemplate:125 → {{ .Code }}
[DEBUG] 11:52:02.723 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 11:52:02.723 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.723 separator.go:Separator() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.723 properties.go:GetString:28 → home_icon: ~
[DEBUG] 11:52:02.723 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 11:52:02.723 separator.go:Separator() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 separator.go:Separator() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.723 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.723 properties.go:GetStringArray:52 → cycle: []
[TRACE] 11:52:02.723 text.go:Render( {{ if .SSHSession }} {{ end }}{{ .UserName }} ) - 530.5µs
[TRACE] 11:52:02.723 text.go:Render(diamond) - 0s
[DEBUG] 11:52:02.724 properties.go:GetString:28 → folder_separator_template:
[DEBUG] 11:52:02.724 properties.go:GetString:28 → folder_separator_icon: \
[DEBUG] 11:52:02.724 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 11:52:02.724 properties.go:GetString:28 → folder_format: %s
[DEBUG] 11:52:02.724 properties.go:GetString:28 → edge_format: %s
[DEBUG] 11:52:02.724 properties.go:GetString:28 → left_format: %s
[DEBUG] 11:52:02.724 properties.go:GetString:28 → right_format: %s
[TRACE] 11:52:02.724 terminal.go:Flags() - 0s
[TRACE] 11:52:02.724 terminal.go:GOOS() - 0s
[TRACE] 11:52:02.724 terminal.go:StackCount() - 0s
[TRACE] 11:52:02.725 text.go:Render({{ .Code }}) - 1.6335ms
[DEBUG] 11:52:02.725 properties.go:GetBool:22 → always_enabled: true
[DEBUG] 11:52:02.725 win32_windows.go:isWriteable:259 → not current user or in group
[DEBUG] 11:52:02.725 win32_windows.go:isWriteable:263 → current user is member of S-1-5-32-544
[DEBUG] 11:52:02.725 win32_windows.go:isWriteable:271 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 11:52:02.725 win32_windows.go:isWriteable:273 → user has write access
[TRACE] 11:52:02.725 terminal_windows.go:DirIsWritable() - 1.038ms
[DEBUG] 11:52:02.725 text.go:patchTemplate:125 →   {{ path .Data.Path .Data.Location }}
[TRACE] 11:52:02.725 text.go:Render(  {{ path .Path .Location }} ) - 0s
[TRACE] 11:52:02.725 text.go:Render(powerline) - 0s
[DEBUG] 11:52:02.727 terminal.go:CommandPath:317 → C:\Users\xxx\AppData\Local\Programs\Git\cmd\git.exe
[TRACE] 11:52:02.727 terminal.go:CommandPath(git.exe) - 4.2857ms
[TRACE] 11:52:02.727 terminal.go:HasCommand(git.exe) - 4.2857ms
[DEBUG] 11:52:02.727 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 11:52:02.727 terminal.go:ResolveSymlink:208 → C:\Users\xxx
[TRACE] 11:52:02.727 terminal.go:ResolveSymlink(C:\Users\xxx) - 0s
[ERROR] 11:52:02.728 terminal.go:HasParentFilePath:476 → CreateFile .git: The system cannot find the file specified.
[TRACE] 11:52:02.728 terminal.go:HasParentFilePath(.git) - 520.9µs
[TRACE] 11:52:02.728 text.go:Render(powerline) - 0s
[TRACE] 11:52:02.728 text.go:Render(  ) - 0s
[TRACE] 11:52:02.728 text.go:Render(powerline) - 0s
[DEBUG] 11:52:02.728 text.go:patchTemplate:125 →  {{ if gt .Code 0 }}{{ else }}{{ end }}
[TRACE] 11:52:02.728 text.go:Render( {{ if gt .Code 0 }}{{ else }}{{ end }} ) - 0s
[DEBUG] 11:52:02.728 text.go:patchTemplate:125 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 11:52:02.728 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 0s
[TRACE] 11:52:02.728 text.go:Render(diamond) - 0s
[DEBUG] 11:52:02.728 text.go:patchTemplate:125 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 11:52:02.728 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 0s
[TRACE] 11:52:02.728 terminal.go:Shell() - 0s
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.728 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.728 segment.go:Execute:112 → segment: Time
[DEBUG] 11:52:02.728 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.728 segment.go:isToggled:229 → no toggles found
[TRACE] 11:52:02.728 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.728 segment.go:Execute:112 → segment: Go
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.728 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.728 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.728 properties.go:GetString:28 → time_format: 15:04:05
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.728 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.728 segment.go:Execute:112 → segment: Python
[DEBUG] 11:52:02.728 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.728 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.728 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.728 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → folders: []
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 11:52:02.728 properties.go:GetString:28 → display_mode: files
[DEBUG] 11:52:02.728 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → folders: []
[DEBUG] 11:52:02.728 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.728 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.728 terminal.go:GOOS() - 0s
[DEBUG] 11:52:02.728 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[DEBUG] 11:52:02.728 segment.go:Execute:112 → segment: Shell
[DEBUG] 11:52:02.728 file.go:Get:70 → cache key not found: toggle_cache
[DEBUG] 11:52:02.728 segment.go:isToggled:229 → no toggles found
[DEBUG] 11:52:02.728 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 11:52:02.728 terminal.go:Shell() - 0s
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[DEBUG] 11:52:02.728 home.go:12 → C:\Users\xxx
[TRACE] 11:52:02.728 text.go:Render(plain) - 0s
[TRACE] 11:52:02.728 text.go:Render(plain) - 0s
[TRACE] 11:52:02.728 text.go:Render(plain) - 0s
[DEBUG] 11:52:02.728 text.go:patchTemplate:125 → in <p:blue><b>{{ .Data.Name }}</b></>
[TRACE] 11:52:02.728 text.go:Render(in <p:blue><b>{{ .Name }}</b></> ) - 0s
[TRACE] 11:52:02.728 text.go:Render(plain) - 0s
[DEBUG] 11:52:02.728 text.go:patchTemplate:125 → at <p:blue><b>{{ .Data.CurrentDate | date "15:04:05" }}</b></>
[TRACE] 11:52:02.728 text.go:Render(at <p:blue><b>{{ .CurrentDate | date "15:04:05" }}</b></>) - 0s
[TRACE] 11:52:02.728 text.go:Render(plain) - 0s
[TRACE] 11:52:02.728 terminal.go:Shell() - 0s
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
[TRACE] 11:52:02.728 terminal.go:Shell() - 0s
[DEBUG] 11:52:02.728 terminal_windows.go:TerminalWidth:99 → terminal width: 120
[TRACE] 11:52:02.728 terminal_windows.go:TerminalWidth() - 0s
[TRACE] 11:52:02.728 path.go:Path() - 0s
[TRACE] 11:52:02.728 terminal.go:Flags() - 0s
```

--- a/src/font/cli.go
+++ b/src/font/cli.go
@@ -177,6 +177,9 @@ func (m *main) Init() tea.Cmd {
 
 	m.progress = progress.NewModel()
 
+	s := spinner.New()
+	m.spinner = &s
+
 	if len(m.URL) != 0 && !isLocalZipFile() {
 		m.state = downloadFont
 
@@ -202,10 +205,8 @@ func (m *main) Init() tea.Cmd {
 		go getFontsList()
 	}()
 
-	s := spinner.New()
-	s.Spinner = spinner.Dot
-	s.Style = lipgloss.NewStyle().Foreground(lipgloss.Color("170"))
-	m.spinner = &s
+	m.spinner.Spinner = spinner.Dot
+	m.spinner.Style = lipgloss.NewStyle().Foreground(lipgloss.Color("170"))
 	m.state = getFonts
 
 	if isLocalZipFile() {

